frontend/public/components/utils/details-item.tsx

```diff
@@ -36,14 +36,28 @@ export const PropertyPath: React.FC<{ kind: string; path: string | string[] }> =
   );
 };
 
-const EditButton: React.SFC<{ onClick: (e: React.MouseEvent<HTMLButtonElement>) => void }> = (
-  props,
-) => (
-  <Button variant="link" isInline onClick={props.onClick}>
-    {props.children}
-    <PencilAltIcon className="co-icon-space-l pf-c-button-icon--plain" />
-  </Button>
-);
+type EditButtonProps = {
+  onClick: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  testId?: string;
+};
+
+const EditButton: React.SFC<EditButtonProps> = (props) => {
+  return (
+    <Button
+      variant="link"
```

Comment:
Not new, but this should have a `type="button"` so it never acts as a form submit button.